hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/CompactingMemStore.java

```diff
@@ -467,7 +498,6 @@ private void doCompact() {
       } catch (Exception e) {
         LOG.debug("Interrupting the MemStore in-memory compaction for store " + getFamilyName());
         Thread.currentThread().interrupt();
```

Comment:
This is actually unchanged code. Should be safe we just do no compaction, because something went wrong... What else should we do?